### DISCUSSION TITLE
有没有更多印章数据集下载链接？能分享一下吗？谢谢

### DISCUSSION THREAD

**Discussion by LMR2018** [PaddlePaddle/PaddleOCR#12650] [Q&A] [answered]
有没有更多印章数据集下载链接？能分享一下吗？谢谢

**Comment by GreatV** [comment, marked as answer]
1. https://github.com/paddlepaddle/PaddleOCR/blob/main/applications/%E5%8D%B0%E7%AB%A0%E5%BC%AF%E6%9B%B2%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB.md
2. https://aistudio.baidu.com/aistudio/datasetdetail/199400
3. https://aistudio.baidu.com/aistudio/datasetdetail/177475

**Reply by LMR2018** [reply]
有没有更多？paddle那点数据不够识别出复杂场景的印章，还有椭圆形印章那些

**Reply by GreatV** [reply]
4. https://github.com/Gmgge/TrOCR-Seal-Recognition/blob/main/DataSet.md

**Reply by LMR2018** [reply]
这些都看过了，有没有更多复杂的，椭圆形的，双环的

**Reply by GreatV** [reply]
这就需要你自己收集制作了

**Reply by LMR2018** [reply]
那有没有什么办法可以快速批量制作呢？

**Reply by ruan2024** [reply]
请问老哥们找到复杂印章数据了嘛，可以分享一下不，我最近也是在弄原形和方形印章检测，找不到数据集。
